README.md

```diff
@@ -259,7 +259,7 @@ You can also use Netdata Parents to:
 
 ### 5. **Connect to Netdata Cloud** :cloud:
 
-[Sign-in](https://app.netdata.cloud/sign-in) to [Netdata Cloud](https://www.netdata.cloud/) and claim your Netdata Agents and Parents.
+[Sign-in](https://app.netdata.cloud/sign-in) to [Netdata Cloud](https://www.netdata.cloud/) and connect your Netdata Agents and Parents.
```

Comment:
> connect your Netdata Agents and Parents

🤔 should we use Children and Parents?

Reply:
Hmmm, we could, albeit the wording is a bit wonky, as both are Agents